contracts/tests/keystone_forwarder.spec.ts

```diff
@@ -83,6 +83,13 @@ describe("keystone_storage", function () {
   });
 
   it("Is initialized!", async () => {
+    // Check event is emitted
+    const listener = program.addEventListener("InitializeEmitEvent", (event, slot) => {
+      assert.isTrue(event.owner.equals(provider.wallet.publicKey), "Owner set")
+      assert.isNotNull(event.authorityNonce)
+      assert.isNotNull(event.authorityNonce)
```

Comment:
event.authorityNonce check is repeated twice here

Reply:
you can actually test for the equality of the actual event.authorityNonce via `actualState.authorithyNonce` below